src/test/java/reposense/parser/ArgsParserTest.java

```diff
@@ -87,6 +89,7 @@ public void parse_withExtraWhitespaces_success() throws ParseException, IOExcept
 
         List<String> expectedFormats = Arrays.asList("java", "adoc", "html", "css", "js");
         Assert.assertEquals(expectedFormats, cliArguments.getFormats());
+
```

Comment:
Same as above, unnecessary line break. Also, add an assertion to check that the `isStandaloneConfigIgnored` is `true`.

Reply:
For now, this flag is only available in LocationCliArgument, so we do not need to add the assertion here.